pyvista/jupyter/pv_pythreejs.py

```diff
@@ -537,6 +537,17 @@ def convert_plotter(pl):
                     width += pv_ren.width + pv_ren.border_width * 2
                 grid[i, j] = convert_renderer(pv_ren)
 
+        # check for linked cameras
+        cameras = [ren.camera for ren in pl.renderers]
+        for cc, camera_a in enumerate(cameras):
+            for dd, camera_b in enumerate(cameras[cc + 1 :]):
+                if camera_a is camera_b:
+                    # flatten indices
+                    ii_source, jj_source = divmod(cc, n_col)
+                    ii_target, jj_target = divmod(dd + 1, n_col)
```

Comment:
And shouldn't this just be `dd`?
```suggestion
                    ii_target, jj_target = divmod(dd, n_col)
```

Reply:
Correct here and above.